stackme-kit/ui/button/button.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type ButtonProps = {
+  /**
+   * a text to be rendered in the component.
+   */
+  text: string
+};
```

Comment:
move this one to types.ts

Reply:
sure, we can move to types.ts but inside package UI components, not global